landingai/pipeline/frameset.py

```diff
@@ -132,7 +132,7 @@ def run_predict(self, predictor: Predictor) -> "Frame":
         ----------
         predictor: the model to be invoked.
         """
-        self.predictions = PredictionList(predictor.predict(np.asarray(self.image)))  # type: ignore
+        self.predictions = PredictionList(predictor.predict(self.image))  # type: ignore
```

Comment:
I was going to comment on that too...it may not be needed any more

Reply:
It is still needed because of other typing confusions in the code that were not solved in this PR (see below). The way we are dealing with every type of prediction (segmentation, classification, OD, OCR, vp, etc) using the same typing classes needs some adjustments in a future PR to account for some differences (not having `label_name` in some cases, having `text` in other cases, etc).

![image](https://github.com/landing-ai/landingai-python/assets/1213561/db2a8098-abfe-466f-9fb7-b07ac22766a9)
![image](https://github.com/landing-ai/landingai-python/assets/1213561/2b211bf0-4d38-42e8-be58-6bb62381e9a7)